replica/table.cc

```diff
@@ -1299,6 +1300,8 @@ table::seal_active_memtable(compaction_group& cg, flush_permit&& flush_permit) n
                     allowed_retries = should_retry(ep) ? default_retries : 0;
                 } else if (auto ep = try_catch<storage_io_error>(ex)) {
                     allowed_retries = should_retry(ep) ? default_retries : 0;
+                } else if (auto ep = try_catch<db::extension_storage_exception>(ex)) {
```

Comment:
i have following compilation error:
```
/home/kefu/dev/scylladb/replica/table.cc:1304:33: error: variable 'ep' set but not used [-Werror,-Wunused-but-set-variable]
 1304 |                 } else if (auto ep = try_catch<db::extension_storage_exception>(ex)) {
      |                                 ^
1 error generated.
```

could you please put something like:
```c++
} else if (try_catch<db::extension_storage_exception>(ex)) {
   /// ...
}
```

Reply:
Done